### PR TITLE
Onboarding: Successful Anserini Reproduction

### PR DESCRIPTION
**Pull request by Linsen-gao-457**
Hi! Thank you for your patience! It's my first time to use github for collaboration... so i have made some mistakes. I hope this won’t cause any inconvenience. And i will show the details below, if there are any other peoblems, plz let me know. Thanks again!

Begin my journey:
ECEUbnuntu (limitted storage)
Operating System: Ubuntu 20.04.6 LTS
Python Version : 3.8.10
Maven Version : 3.8.1
Memory: 125 GiB(only 8GB for ECE students)
Outcome: All steps completed successfully

Second project:
My window computer
Operating System: Windows 10 Home Chinese Version
Java Version: 21.0.4 2024-07-16 LTS
Apache Maven Version: 3.9.9
Python Version : 3.8.10
Outcome: All steps completed successfully

Issues Encountered:
2 problems - mvn clean package
1. Windows Virtual Memory Allocation Issue:
High memory usage during inverted index construction caused system memory exhaustion.
Solution: Manually increased Windows virtual memory allocation to approximately 200 GB to ensure the indexing process completed smoothly.
2. I use mvn clean package -DskipTests instead of -Dmaven.test.skip=true, but mvn clean package "-Dmaven.test.skip=true" can work to install Anserini. The reason -DskipTests and -Dmaven.test.skip=true work seamlessly while -Dmaven.test.skip=true may fail in PowerShell is due to how Maven and the terminal handle arguments:

Another problem - TREC Tool Incompatibility on Windows: The default trec_eval tool does not support native Windows execution.
Solution: Recompiled trec_eval using MSYS2 to successfully execute the tool

### CODE REVIEW

File: docs/experiments-msmarco-passage.md [hunk 1]
```diff
@@ -541,4 +541,5 @@ The BM25 run with default parameters `k1=0.9`, `b=0.4` roughly corresponds to th
 + Results reproduced by [@nihalmenon](https://github.com/nihalmenon) on 2024-11-29 (commit [`aa91e2db`](https://github.com/castorini/anserini/commit/aa91e2db224c752cc6a947bdf66bdbf9ce90af25))
 + Results reproduced by [@sherloc512](https://github.com/sherloc512) on 2024-12-04 (commit [`9e55b1c`](https://github.com/castorini/anserini/commit/9e55b1c97fced46530dac1f78975d19635ffaf7a))
 + Results reproduced by [@zdann15](https://github.com/zdann15) on 2024-12-04 (commit [`9d311b4`](https://github.com/castorini/anserini/commit/9d311b4409a9ff3d79b01910178eaec3931f0abe))
-+ Results reproduced by [@Alireza-Zwolf](https://github.com/Alireza-Zwolf) on 2024-12-15 (commit [`c7dff5f`](https://github.com/castorini/anserini/commit/c7dff5f8417905612ad9f97e85012440e9e16087))
\ No newline at end of file
++ Results reproduced by [@Alireza-Zwolf](https://github.com/Alireza-Zwolf) on 2024-12-15 (commit [`c7dff5f`](https://github.com/castorini/anserini/commit/c7dff5f8417905612ad9f97e85012440e9e16087))
++ Results reproduced by [@Forrest](https://github.com/Linsen-gao-457) on 2024-12-17 (commit [a86484a6](https://github.com/castorini/anserini/commit/a86484a6e99a7a97966c423d230ad05279b24508))
```
Comment (lintool) [inline docs/experiments-msmarco-passage.md:545]: Please change @Forrest to @Linsen-gao-457 , your actual user name?

**Review by lintool — CHANGES_REQUESTED** [review]

File: docs/experiments-msmarco-passage.md (hunk 1 above)
Comment (Linsen-gao-457) [reply docs/experiments-msmarco-passage.md:545]: OK!

Comment (Linsen-gao-457) [reply docs/experiments-msmarco-passage.md]: > Please change @forrest to @Linsen-gao-457 , your actual user name?

Already update! Is that ok now? or i need to do a new pull request

Comment (lintool) [reply docs/experiments-msmarco-passage.md]: all good, waiting for tests to pass before I merge

Comment (Linsen-gao-457) [reply docs/experiments-msmarco-passage.md]: "Got it! thank you very much!!

**Review by lintool — APPROVED** [review]

**Pull request merged**